websites/R/RTL+/presence.ts

```diff
@@ -0,0 +1,149 @@
+const presence = new Presence({
+		clientId: "1033504954763198545",
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let title: string,
+	currentTime: number,
+	video: HTMLVideoElement,
+	duration: number,
+	paused: boolean;
+
+presence.on("UpdateData", async () => {
+	const [startTimestamp, endTimestamp] = presence.getTimestamps(
+			Math.floor(currentTime),
+			Math.floor(duration)
+		),
+		presenceData: PresenceData = {
+			largeImageKey: "logo",
```

Comment:
Imgur links are preferred over Discord asset names
```suggestion
			largeImageKey: "https://i.imgur.com/fMMsZfV.png",
```